src/includes/enriching-events/scopes/local-scope/android.mdx

```diff
@@ -0,0 +1 @@
+The Sentry Android SDK doesn't support the concept of `Local Scopes`. A single `Scope` is used everwhere in the App's lifecycle.
```

Comment:
Ok. My goal here was simply to clarify state as I understood it from the existing docs. Currently the Android docs tell you that Local Scopes can't be used and then proceed to tell you how to use them anyways, I thought it would be more clear to just leave it at saying that they can't be used. See my comment [here](https://github.com/getsentry/sentry-docs/pull/4747/files#r816456573) for a comparison of screenshots.

Reply:
I'd go ahead with this change since it just improves the current state of the SDK, once we land https://github.com/getsentry/sentry-java/issues/1829
We can work on docs again.